README.md

```diff
@@ -39,22 +34,22 @@ This release is supported by the next SDK versions
 
 - azure-iot-sdk-c [public-preview](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp) branch
```

Comment:
Could you please call it the `public-preview-pnp` and not just `public-preview` in human readable section.  The link itself is correct.

Reply:
fixing. (I assume we will merge `public-preview-pnp` into `public-preview` by 5/19. Requiring us to update the docs by that date).

Reply:
Yes to public-preview by 5/19.